tests/publiccloud/transfer_repos.pm

```diff
@@ -17,13 +17,20 @@ use warnings;
 use testapi;
 use strict;
 use utils;
+use publiccloud::utils "select_host_console";
 
 sub run {
     my ($self, $args) = @_;
 
     my @addons = split(/,/, get_var('SCC_ADDONS', ''));
 
-    select_console 'tunnel-console';
+    select_host_console();    # select console on the host, not the PC instance
+
+    # Shortcut for not doing updates (useful for speedup of verification runs)
+    if (get_var('PUBLIC_CLOUD_SKIP_DOWNLOAD') == 1) {
```

Comment:
Just a nit hint and yes it's also not visible (at least not with a message), but we could achieve the same for **debugging** with
`EXCLUDE_MODULES=transfer_repos` , or?

Reply:
That SHOULD work as well, indeed. Or more specifically `EXCLUDE_MODULES=download_repos,transfer_repos`

Reply:
The new test runs work this `EXCLUDE_MODULES` as well, so I consider this as resolved now